# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderBridgeParent.h
# Commit: 971767c6ed2b
# Full Hash: 971767c6ed2b2201e172105046ecc61942e03a7e
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2020-01-24 21:46:30
# Regressor Bug: 1602643
# File Overlap Count: 1
# Description:
#   Bug 1602643 - Disable WebRender compositor dinamically for async screenshot r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D59658
# ==============================================================================

diff -r 436df7dc06c6 -r 971767c6ed2b gfx/layers/wr/WebRenderBridgeParent.h
--- a/gfx/layers/wr/WebRenderBridgeParent.h	Fri Jan 24 01:04:09 2020 +0000
+++ b/gfx/layers/wr/WebRenderBridgeParent.h	Thu Jan 23 23:52:05 2020 +0000
@@ -317,6 +317,8 @@
    */
   RefPtr<wr::WebRenderAPI::GetCollectedFramesPromise> GetCollectedFrames();
 
+  void DisableNativeCompositor();
+
  private:
   class ScheduleSharedSurfaceRelease;
 